cli/lsp/documents.rs

```diff
@@ -114,59 +118,39 @@ impl DocumentData {
     &mut self,
     content_changes: Vec<lsp::TextDocumentContentChangeEvent>,
   ) -> Result<(), AnyError> {
-    if self.bytes.is_none() {
-      return Ok(());
-    }
-    let content = &mut String::from_utf8(self.bytes.clone().unwrap())
-      .context("unable to parse bytes to string")?;
```

Comment:
It was being stored as bytes because that is what rust-analyzer did, and I didn't know how we were going to deal with it and handle it, so it was best to copy that pattern and figure out challenges later.